Clean package xml and set version to 0.2.0

JeroenDM opened the pull request:
#40

JeroenDM committed:
add author and version 0.2.0

simonschmeisser commented on package.xml:
```diff
@@ -1,13 +1,22 @@
+  <!-- <url type="website">http://wiki.ros.org/opw_kinematics_plugin</url> -->
+
+  <author>Jeroen De Maeyer</author>
+  <author>Simon Schmeisser</author>
```
```suggestion
  <author>Simon Schmeisser (isys vision)</author>
```

simonschmeisser commented on package.xml:
```diff
@@ -1,13 +1,22 @@
+  <version>0.2.0</version>
+  <description>
+    <p>MoveIt kinematics plugin for industrial robots.</p>
+    <p> This plugin uses an analytical inverse kinematic library, opw_kinematics, to calculate the inverse kinematics for industrial robots with 6 degrees of freedom, two parallel joints, and a spherical wrist.</p>
```
not sure if there are any rules about line lengths but you could add a line break here somewhere

JeroenDM committed:
Add affiliations in package.xml

Co-Authored-By: Simon Schmeisser <simon.schmeisser@isys-vision.de>

JeroenDM committed:
Add line breaks in package.xml

based on example in fanuc repo: https://github.com/ros-industrial/fanuc/blob/indigo-devel/fanuc_resources/package.xml

JeroenDM commented:
@simonschmeisser  thank you for the review.
